packages/vite/src/node/optimizer/scan.ts

```diff
@@ -203,18 +203,12 @@ function esbuildScanPlugin(
         external: true
       }))
 
-      build.onResolve(
-        { filter: virtualModuleRE },
-        async ({ path, importer }) => {
-          return {
-            path: await resolve(
-              path.substring('virtual-module:'.length),
-              importer
-            ),
```

Comment:
To further elaborate, `resolve()` here would result in something like: `/home/bjorn/path/to/comp.svelte`...